File: all-submissions/0782-jewels-and-stones/solution.java
class Solution {
    public int numJewelsInStones(String J, String S) {
        HashSet<String> jewels = new HashSet<String>();
        int count = 0;
        String[] jewel = J.split("");
        for(int i = 0; i<jewel.length; i++)
        {
            jewels.add(jewel[i]);
        }
        String[] stone = S.split("");
        for(int j = 0;j<stone.length;j++)
        {
            if(jewels.contains(stone[j]))
            {
                count++;
            }
        }
        return count;
    }
}

File: all-submissions/0782-jewels-and-stones/solution.py
class Solution:
    def numJewelsInStones(self, J: str, S: str) -> int:
        sum = 0 
        for i in range (0, len(J)): 
            for j in range (0, len(S)): 
                if J[i] == S[j]: 
                    sum = sum + 1 
        return sum 
